### PR TITLE
Update permissions.yml zeus

### PR DESCRIPTION
**Pull request by flamingwarrior10**
donno if it is done

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -4753,6 +4753,71 @@ groups:
...
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.run6
+        - ch.alias.run7
+        - ch.alias.run8
+        - ch.alias.kill
+        - ch.alias.lightning
+        - ch.alias.gmc
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: requires evidence that he can manipulate matter

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -4753,6 +4753,71 @@ groups:
...
+        - ch.alias.gms
+        - libsdisguises.disguise
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: require evidence that he can disguise

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -4753,6 +4753,71 @@ groups:
...
+        - ch.alias.run0
+        - ch.alias.run1
+        - ch.alias.run2
+        - ch.alias.run3
+        - ch.alias.run4
+        - ch.alias.run5
+        - ch.alias.run6
+        - ch.alias.run7
+        - ch.alias.run8
+        - ch.alias.kill
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: need evidence of killing at will

Comment (flamingwarrior10) [inline PermissionsEx/permissions.yml]: https://en.wikipedia.org/wiki/Shapeshifting
ik my greek stuff bro

Comment (flamingwarrior10) [inline PermissionsEx/permissions.yml]: i have read the Illiad and Odyssey along with many other myths

**Pull request merged**